apps/ui/src/components/Ui/Markdown.vue

```diff
@@ -36,13 +54,42 @@ const parsed = computed(() => {
 
   return remarkable.render(formattedBody);
 });
+
+onMounted(() => {
+  const body = document.querySelector('.markdown-body');
+  if (body !== null) {
```

Comment:
If we just flip it then we can do `if (!body) return` and code below is cleaner.